### ISSUE TITLE
openness 20.12  harbor pod connect harbor-db failed 

### ISSUE THREAD

**Issue by liuuwqia** [open-ness/converged-edge-experience-kits#88]
env: centos7.8
openness:20.12
kubernets:1.19.3
cni: ovn4nfv

ansible.log
Internal tls trust CA appending is Done.
2021-02-22T04:31:08Z [INFO] [/controller/artifact/annotation/parser.go:71]: the annotation parser to parser artifact annotation version v1alpha1 registered
2021-02-22T04:31:08Z [INFO] [/controller/artifact/processor/processor.go:59]: the processor to process media type application/vnd.cncf.helm.config.v1+json registered
2021-02-22T04:31:08Z [INFO] [/controller/artifact/processor/processor.go:59]: the processor to process media type application/vnd.cnab.manifest.v1 registered
2021-02-22T04:31:08Z [INFO] [/controller/artifact/processor/processor.go:59]: the processor to process media type application/vnd.oci.image.index.v1+json registered
2021-02-22T04:31:08Z [INFO] [/controller/artifact/processor/processor.go:59]: the processor to process media type application/vnd.docker.distribution.manifest.list.v2+json registered
2021-02-22T04:31:08Z [INFO] [/controller/artifact/processor/processor.go:59]: the processor to process media type application/vnd.docker.distribution.manifest.v1+prettyjws registered
2021-02-22T04:31:08Z [INFO] [/controller/artifact/processor/processor.go:59]: the processor to process media type application/vnd.oci.image.config.v1+json registered
2021-02-22T04:31:08Z [INFO] [/controller/artifact/processor/processor.go:59]: the processor to process media type application/vnd.docker.container.image.v1+json registered
2021-02-22T04:31:08Z [INFO] [/replication/adapter/native/adapter.go:36]: the factory for adapter docker-registry registered
2021-02-22T04:31:08Z [INFO] [/replication/adapter/harbor/adaper.go:31]: the factory for adapter harbor registered
2021-02-22T04:31:08Z [INFO] [/replication/adapter/dockerhub/adapter.go:25]: Factory for adapter docker-hub registered
2021-02-22T04:31:08Z [INFO] [/replication/adapter/huawei/huawei_adapter.go:27]: the factory of Huawei adapter was registered
2021-02-22T04:31:08Z [INFO] [/replication/adapter/googlegcr/adapter.go:35]: the factory for adapter google-gcr registered
2021-02-22T04:31:08Z [INFO] [/replication/adapter/awsecr/adapter.go:47]: the factory for adapter aws-ecr registered
2021-02-22T04:31:08Z [INFO] [/replication/adapter/azurecr/adapter.go:15]: Factory for adapter azure-acr registered
2021-02-22T04:31:08Z [INFO] [/replication/adapter/aliacr/adapter.go:31]: the factory for adapter ali-acr registered
2021-02-22T04:31:08Z [INFO] [/replication/adapter/jfrog/adapter.go:47]: the factory of jfrog artifactory adapter was registered
2021-02-22T04:31:08Z [INFO] [/replication/adapter/quay/adapter.go:41]: the factory of Quay adapter was registered
2021-02-22T04:31:08Z [INFO] [/replication/adapter/helmhub/adapter.go:30]: the factory for adapter helm-hub registered
2021-02-22T04:31:08Z [INFO] [/replication/adapter/gitlab/adapter.go:17]: the factory for adapter gitlab registered
2021-02-22T04:31:08Z [INFO] [/core/controllers/base.go:299]: Config path: /etc/core/app.conf
2021-02-22T04:31:08Z [INFO] [/core/main.go:162]: initializing configurations...
2021-02-22T04:31:08Z [INFO] [/core/config/config.go:84]: key path: /etc/core/key
2021-02-22T04:31:08Z [INFO] [/core/config/config.go:61]: init secret store
2021-02-22T04:31:08Z [INFO] [/core/config/config.go:64]: init project manager
2021-02-22T04:31:08Z [INFO] [/core/config/config.go:96]: initializing the project manager based on local database...
2021-02-22T04:31:08Z [INFO] [/core/main.go:164]: configurations initialization completed
2021-02-22T04:31:08Z [INFO] [/common/dao/base.go:65]: Registering database: type-PostgreSQL host-harbor-app-harbor-database port-5432 database-registry sslmode-"disable"
2021-02-22T04:31:09Z [ERROR] [/common/utils/utils.go:106]: failed to connect to tcp://harbor-app-harbor-database:5432, retry after 2 seconds :dial tcp: i/o timeout
2021-02-22T04:31:13Z [ERROR] [/common/utils/utils.go:106]: failed to connect to tcp://harbor-app-harbor-database:5432, retry after 2 seconds :dial tcp: i/o timeout
2021-02-22T04:31:19Z [ERROR] [/common/utils/utils.go:106]: failed to connect to tcp://harbor-app-harbor-database:5432, retry after 2 seconds :dial tcp: i/o timeout
2021-02-22T04:31:29Z [ERROR] [/common/utils/utils.go:106]: failed to connect to tcp://harbor-app-harbor-database:5432, retry after 2 seconds :dial tcp: i/o timeout
2021-02-22T04:31:47Z [ERROR] [/common/utils/utils.go:106]: failed to connect to tcp://harbor-app-harbor-database:5432, retry after 2 seconds :dial tcp: i/o timeout
2021-02-22T04:32:21Z [ERROR] [/common/utils/utils.go:106]: failed to connect to tcp://harbor-app-harbor-database:5432, retry after 2 seconds :dial tcp: i/o timeout

**Comment by archie951** [comment]
Hi @liuuwqia, You can try enabling both calico and ovn4nfv in group_vars/all/10-default.yml. Calico will go first and then ovn4nfv. Let me know if that works.

**Comment by liuuwqia** [comment]
Hi archie951

Calico is ok, ovn4nfv need to be tested, Did you submit new code?

**Comment by archie951** [comment]
No. I did not submit new code. I was testing with tag 20.12.02. I am also facing issues with ovn4nfv as standalone cni.

**Comment by liuuwqia** [comment]
I just tester calico to do cni alone ,but ovn4nfv  not success, I check the harbor log, maybe not harbor problem ,I think ovn4nfv cni maybe have problem probably

But  when i use yaml file deployed calico and ovn4nfv in k8s platform ,it`s ok.
